### ISSUE TITLE
support mp3 and opus files for playback

### ISSUE THREAD

**Issue by wolftune** [essej/sonobus#32]
So, I tried loading audio files to play, and ogg and wav worked but mp3 and opus files failed.

Lack of opus support especially seems odd given using opus otherwise.

**Comment by essej** [comment]
Yeah, I'll look into adding MP3 support for reading at least, now that all the patents are expired. As for missing Opus file format, it is ironic... it's just that opus as a file format hasn't picked up a lot of momentum yet, but I should be able to get it in there someday.

**Comment by wolftune** [comment]
I wish opus took off as a file format, but one place it shows up a lot: the audio only `-x` use of youtube-dl

**Comment by Zipdox** [comment]
Can you not just use a library for decoding media files? Like libavcodec and libavformat?

**Comment by ericfont** [comment]
I'd second using libavcodec, cause then that would cover support for importing all sorts of random formats.
